docker-compose.gcp.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+services:
+  http-api:
+    logging:
+      driver: fluentd
+      options:
+        fluentd-address: 127.0.0.1:24224
+        tag: docker.http_api
+    volumes:
+      - /var/opt/radicle-seed/.radicle:/app/radicle:ro
+    user: $SEED_USER_ID:$SEED_USER_ID
+    command: ["--log-format", "gcp"]
+  org-node:
+    logging:
+      driver: fluentd
+      options:
+        fluentd-address: 127.0.0.1:24224
+        tag: docker.org_node
+    volumes:
+      - /var/opt/radicle-seed/.radicle:/app/radicle
+    user: $SEED_USER_ID:$SEED_USER_ID
+    command: ["--subgraph", "https://gateway.thegraph.com/api/1758a78ae257ad4906f9c638e4a68c19/subgraphs/id/0x2f0963e77ca6ac0c2dad1bf4147b6b40e0dd8728-0", "--orgs", "0x8152237402e0f194176154c3a6ea1eb99b611482", "--rpc-url", "wss://eth-mainnet.alchemyapi.io/v2/RDOfd2g0-_-KEOh7LwBCcovkVRM2QI2m", "--log-format", "gcp"]
```

Comment:
The RPC URL should be taken out of here, as it contains a private api token.

Reply:
On the CLI we use an env var called `ETH_RPC_URL`, maybe we can use the same here?

Reply:
OK, done.  I've put `ETH_RPC_URL` into `.env` in an encrypted form.  You may want to take a look at [sops](https://github.com/mozilla/sops#examples).  It very easy to use.  I've added the command to decrypt to the README file.